search-commons/src/main/java/no/unit/nva/search/SearchClient.java

```diff
@@ -136,25 +87,26 @@ private SearchResponse doSearch(String term,
 
     }
 
-    private SearchRequest getSearchRequest(String term, int results, int from, String orderBy, SortOrder sortOrder) {
+    private SearchRequest getSearchRequest(
+            String searchTerm, int results, int from, String orderBy, SortOrder sortOrder) {
         final SearchSourceBuilder sourceBuilder = new SearchSourceBuilder()
-                .query(QueryBuilders.queryStringQuery(term))
+                .query(QueryBuilders.queryStringQuery(searchTerm))
                 .sort(orderBy, sortOrder)
                 .from(from)
                 .size(results);
         return new SearchRequest(ELASTICSEARCH_ENDPOINT_INDEX).source(sourceBuilder);
```

Comment:
I assume that this will change in later PR and be dynamic, so that we can serve all indexes?

Reply:
Hm, good point. I can do it now.